bp/src/BPConfiguration.m

```diff
@@ -332,6 +336,9 @@ - (NSString *)configString {
     if (self.environmentVariables) {
         [dict setValue:self.environmentVariables forKey:@"environmentVariables"];
     }
+    if (self.dependencies) {
+        [dict setValue:self.dependencies forKey:@"dependencies"];
+    }
```

Comment:
Since we have a default value for testApplicationDependencies, this check is probably redundant. Let me know what you think.